### DISCUSSION TITLE
Streaming app, skipping or missing messages or something worse...

### DISCUSSION THREAD

**Discussion by bmmptlgc** [LGouellec/streamiz#336] [General]
Hi, I created a streaming app that reads messages from a multi schema topic and sends each message type to it's own topic. Here's the repo: https://github.com/bmmptlgc/TopicSplitter/tree/master

The example linked publishes 1000 OrderCreated messages and 1000 OrderCompleted messages to the `orders` topic. The streaming app will consume those message and I would expect 2 new topics to be created, each with 1000 messages. But the app is very moody... Sometimes it doesn't send any messages to new topics, but the offsets for the consumer group on the original topic seem to be committed. Sometimes it only processes part of the messages as the screenshot below shows:
![image](https://github.com/LGouellec/kafka-streams-dotnet/assets/109662343/baefcb74-d045-435f-b191-f2389853d48f)

Any help would be appreciated.

**Comment by bmmptlgc** [comment]
I believe I was missing `AutoOffsetReset = AutoOffsetReset.Earliest`.

**Reply by LGouellec** [reply]
So I considered your issue fixed ?
